src/apps/frontend/components/layouts/task-item.layout.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { PropsWithChildren } from 'react';
+
+const TaskItemLayout: React.FC<PropsWithChildren> = ({ children }) => (
```

Comment:
Why is TaskItem a common component i.e. why is it under `components/layouts` folder?

Reply:
Basically, TaskItem Layouts are used for adding multiple tasks which contain a title and description, so I made a common layout to add the tasks that fall under TaskItem Layouts.

Reply:
Why is it under `common` layout?

The way to design components are:

1. If its generic layout, then it should be named as such VeritcalStack etc.
2. If its not generic layout, it should not be in components folder